.travis/run_tests.sh

```diff
@@ -21,7 +21,7 @@ if [ ! -z ${changed_files} ]; then
 		if [ ! -f ${f} ]; then
 			continue
 		fi
-		ts_name=$(grep -oP 'class[\s]+(\K\w+)(?=\(.*\):)' ${f})
+		ts_name=$(grep -oP 'class[\s]+(\KTest\w+)(?=\(.*\):)' ${f})
```

Comment:
@arungrover Again nice catch! I agree we should ignore \_\_init__.py, will change code.